common/core/nesteddisplay.cpp

```diff
@@ -67,10 +170,110 @@ bool NestedDisplay::SetPowerMode(uint32_t /*power_mode*/) {
+
+        /* TODO: add more flexibility here, instead of hardcoded domain 0*/
+        msg.remote_domain = 0;
+        msg.dmabuf_fd = buffer->GetPrimeFD();
+
+        ret = ioctl(mHyperDmaBuf_Fd, IOCTL_HYPER_DMABUF_EXPORT_REMOTE, &msg);
+        if (ret) {
+          ETRACE("Hyper DmaBuf: Exporting hyper_dmabuf failed with error %d\n", ret);
+          return false;
+        }
+        else {
+          ETRACE("Hyper DmaBuf: Exporting hyper_dmabuf Done! 0x%x\n", msg.hid.id);
+          uint32_t HalFormat = DrmFormatToHALFormat(buffer->GetFormat());
+
+          mHyperDmaExportedBuffers[sf_handle].width = buffer->GetWidth();
+          mHyperDmaExportedBuffers[sf_handle].height = buffer->GetHeight();
+          switch (HalFormat) {
+            case HAL_PIXEL_FORMAT_RGBX_8888:
+            case HAL_PIXEL_FORMAT_RGB_888:
+            case HAL_PIXEL_FORMAT_RGB_565:
+              mHyperDmaExportedBuffers[sf_handle].format = EGL_TEXTURE_RGB;
+              break;
+            case HAL_PIXEL_FORMAT_RGBA_8888:
+            case HAL_PIXEL_FORMAT_BGRA_8888:
+              mHyperDmaExportedBuffers[sf_handle].format = EGL_TEXTURE_RGBA;
+              break;
+            case HAL_PIXEL_FORMAT_NV12_Y_TILED_INTEL:
+              mHyperDmaExportedBuffers[sf_handle].format = 0x31D8;        //this wayland specified format, hardcode here.
+              break;
+          }
+          pitches = buffer->GetPitches();
+          offsets = buffer->GetOffsets();
+          mHyperDmaExportedBuffers[sf_handle].pitch[0] = pitches[0];
+          mHyperDmaExportedBuffers[sf_handle].pitch[1] = pitches[1];
+          mHyperDmaExportedBuffers[sf_handle].pitch[2] = pitches[2];
+          mHyperDmaExportedBuffers[sf_handle].offset[0] = offsets[0];
+          mHyperDmaExportedBuffers[sf_handle].offset[1] = offsets[1];
+          mHyperDmaExportedBuffers[sf_handle].offset[2] = offsets[2];
+          mHyperDmaExportedBuffers[sf_handle].bpp = bitsPerPixel(HalFormat);
+          mHyperDmaExportedBuffers[sf_handle].tile_format = buffer->GetTilingMode();
+          mHyperDmaExportedBuffers[sf_handle].rotation = 0;;
+          mHyperDmaExportedBuffers[sf_handle].status = 0;
+          mHyperDmaExportedBuffers[sf_handle].counter = 0;
+          mHyperDmaExportedBuffers[sf_handle].hyper_dmabuf_id = msg.hid;
+          mHyperDmaExportedBuffers[sf_handle].surface_id = (uint64_t)sf_handle;
+          strncpy(mHyperDmaExportedBuffers[sf_handle].surface_name,"Cluster",SURFACE_NAME_LENGTH);
+          mHyperDmaExportedBuffers[sf_handle].bbox[0] = display_frame.left;
+          mHyperDmaExportedBuffers[sf_handle].bbox[1] = display_frame.top;
+          mHyperDmaExportedBuffers[sf_handle].bbox[2] = buffer->GetWidth();
+          mHyperDmaExportedBuffers[sf_handle].bbox[3] = buffer->GetHeight();
+        }
+      }
+    }
+    memcpy(buf+sizeof(int)+header_size+info_size*buffer_number,&mHyperDmaExportedBuffers[sf_handle],info_size);
```

Comment:
please indent, check coding style: https://github.com/intel/IA-Hardware-Composer/wiki/Contributions

Reply:
ok, will use clang-format-diff regenerate all the code.